### PR TITLE
Training skip list should not be applied on inference bench

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #104738

cc @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @ipiszy @chenyang78

### CODE REVIEW

File: benchmarks/dynamo/torchbench.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def setup_torchbench_cwd():
     # TIMEOUT, https://github.com/pytorch/pytorch/issues/98467
     "tacotron2",
 }
+SKIP.update(DETECTRON2_MODELS)
```
Comment (BowenBao) [inline benchmarks/dynamo/torchbench.py:80]: https://github.com/pytorch/pytorch/actions/runs/5480430769/jobs/9984339567

Comment (BowenBao) [reply benchmarks/dynamo/torchbench.py:80]: https://github.com/facebookresearch/detectron2/commit/ff53992b1985b63bd3262b5a36167098e3dada02 should resolve this.

**Review by thiagocrepaldi — APPROVED** [review]

File: benchmarks/dynamo/torchbench.py (hunk 1 above)
Comment (desertfire) [reply benchmarks/dynamo/torchbench.py:80]: Please do follow up on this. Thanks!

**Review by desertfire — APPROVED** [review]

File: benchmarks/dynamo/torchbench.py (hunk 1 above)
Comment (BowenBao) [reply benchmarks/dynamo/torchbench.py:80]: Or after #104760

**Pull request closed without merging**